composeApp/src/commonMain/kotlin/com/github/familyvault/forms/GroupChatCreationForm.kt

```diff
@@ -0,0 +1,64 @@
+package com.github.familyvault.forms
+
+import com.github.familyvault.forms.validator.FormValidator
+import com.github.familyvault.forms.validator.FormValidatorError
+import com.github.familyvault.models.FamilyMember
+import com.github.familyvault.models.forms.FormDataListEntry
+import com.github.familyvault.models.forms.FormDataStringEntry
+
+data class GroupChatCreationFormData(
+    val groupName: FormDataStringEntry = FormDataStringEntry(),
+    var familyMembers: FormDataListEntry<FamilyMember> = FormDataListEntry(),
+)
+
+class GroupChatCreationForm: BasicForm() {
```

Comment:
Docelowo zamierzamy reużyć tworzenie grup chatu z edycją, więc może nazwijmy to `GroupChatEditForm`